packages/integration_test/ios/Classes/IntegrationTestPlugin.m

```diff
@@ -1,6 +1,6 @@
 #import "IntegrationTestPlugin.h"
 
-static NSString *const kIntegrationTestPluginChannel = @"plugins.flutter.io/integratoin_test";
```

Comment:
I could have sworn @bparrishMines fixed this previously.

Reply:
Sorry, I was supposed to. I got distracted while waiting for our CI to go green.